packages/@react-aria/overlays/src/usePopover.ts

```diff
@@ -112,6 +113,14 @@ export function usePopover(props: AriaPopoverProps, state: OverlayTriggerState):
     }
   }, [isNonModal, state.isOpen, popoverRef]);
 
+  useLayoutEffect(() => {
+    if (state.isOpen && !isPositioned) {
+      setDelayScrolling(true);
+    } else {
+      setDelayScrolling(false);
+    }
```

Comment:
if it unmounts while open, we probably need to do cleanup?

Reply:
Ah, good catch, I suppose this could render but unmount before positioning finishes.